Bitrise/testing_bitrise.yml

```diff
@@ -93,7 +93,7 @@ workflows:
             fi
 
             # Get the latest master branch for WeTransfer-iOS-CI if the submodule exists
-            git submodule update --remote --no-fetch Submodules/WeTransfer-iOS-CI
+            # git submodule update --remote --no-fetch Submodules/WeTransfer-iOS-CI
```

Comment:
This is only to allow Mule testing: https://github.com/WeTransfer/Mule/pull/2201 and will be reverted before merging